### DISCUSSION TITLE
Best models to separate movies/animes?

### DISCUSSION THREAD

**Discussion by JestAK** [Anjok07/ultimatevocalremovergui#1267] [Q&A]
Even if I have 6 channel there is still important effects on channel with actor's dubbing.

**Which models and settings will manage separation voices from sfx and background music the best with minimum artifacts?**
If you experienced movie or anime separation, can you share your results?

**Comment by Jellybit** [comment]
I'm trying to figure this out myself as well. I feel like this would be a common use. I'll continue experimentation.
